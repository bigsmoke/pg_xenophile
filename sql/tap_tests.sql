begin;

create extension pg_xenophile
    with cascade;

call xeno.test__l10n_table();

rollback;
