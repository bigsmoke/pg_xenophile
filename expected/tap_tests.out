begin;
create extension pg_xenophile
    with cascade;
NOTICE:  installing required extension "hstore"
call xeno.test__l10n_table();
NOTICE:  drop cascades to 3 other objects
DETAIL:  drop cascades to view test_tbl_l10n_en
drop cascades to view test_tbl_l10n_fr
drop cascades to view test_tbl_l10n_nl
NOTICE:  drop cascades to 3 other objects
DETAIL:  drop cascades to view test_tbl_l10n_en
drop cascades to view test_tbl_l10n_fr
drop cascades to view test_tbl_l10n_nl
NOTICE:  drop cascades to 4 other objects
DETAIL:  drop cascades to constraint test_tbl_l10n_id_fkey on table test_tbl_l10n
drop cascades to view test_tbl_l10n_en
drop cascades to view test_tbl_l10n_fr
drop cascades to view test_tbl_l10n_nl
rollback;
